src/core/src/energy/pairs.rs

```diff
@@ -147,6 +147,11 @@ impl PairInteraction {
     pub fn set_restriction(&mut self, restriction: PairRestriction) {
         self.restriction = restriction;
     }
+
+    /// Return the cutoff radius
+    pub fn get_cutoff(&self) -> f64 {
```

Comment:
As this is a public function, can you add an example here? See the other functions or Rust standard library for example of `examples` 😄 The example should use this function and have at least one `assert_eq!` for checking the function result.